components/widgets/menu/menu_items/menu_item.scss

```diff
@@ -56,7 +56,7 @@
                 background-color: var(--center-channel-color-08) !important;
             }
         }
-
+        // TODO: add classes to avoid conflicts
```

Comment:
What's this TODO for? Is this a follow-up ticket or something you intend to solve now?

Reply:
Ideally a follow-up ticket where the scope is known in the team as its going to be large. I would like to keep the comments and maybe add more whenever we see things that can be improved so we can refactor consistently.

Reply:
Okay, can you create the ticket and add it to this comment?